src/main/markdown/doc/latest/DevGuideValidation.md

```diff
@@ -1,6 +1,8 @@
-Validation
+Validation (DEPRECATED)
 ===
 
+**Attention this API is deprecated in GWT 2.8 and will be removed in future versions**
```

Comment:
You could say, removed from gwt and add a note that there is going to be a spereate project for validation